filetests/regalloc/x86-regres.clif

```diff
@@ -24,17 +25,19 @@ ebb2(v4: i32, v5: i32, v7: i32):
     v8 = iconst.i32 -1
     ; v7 is killed here and v9 gets the same register.
     v9 = iadd v7, v8
-    ; check: v9 = iadd v7, v8
+    ; check:   v9 = iadd v7, v8
```

Comment:
Why these extra space? `check` should trim the input.

Reply:
This is to have a 2 spaces indentation of instructions compared to basic blocks labels. These should not matter when testing.